### PR TITLE
Add hamburger toogle menu functionality

### PR DESCRIPTION
**Pull request by Thermiee**
- When the user clicks (or taps) the hamburger button, the mobile menu appears.
- When the user clicks (or taps) the close (X) button, the mobile menu disappears.
- When the user clicks (or taps) any of the mobile menu options, the mobile menu disappears.
- When the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2,334 +2,298 @@
...
-            </li>
-            <li class="social-media">
-                <img src="images/fingers.png" alt="two fingers">
-            </li>
+  <header>
+    <div class="container">
+      <nav class="nav-item1">My Logo</nav>
+      <div class="nav-item3">
+        <span class="hamburger-span" onclick="openNav()"> &#9776; </span>
+      </div>
```
Comment (andersonlebon) [inline index.html]: - [ ] Nice job 👏  👏  On implementing the mobile menu functionality I can see it is working properly, however one of the requirements says that you should use Javascript best practices. I think your project will be well structured if you put all Javascript functionality(events) in your `main.js file`. Kindly remove this `onclick` event here in your HTML and use `addEventListener` in `main.js`
#### An example in your main.js
```js
    document.querySelector('hamburger-span').addEventListener('click', openNav)
```

File: index.html [hunk 2]
```diff
@@ -2,334 +2,298 @@
...
-                </div>
+      </nav>
+    </div>
+    <div id="myNav" class="overlay">
+      <a href="javascript:void(0)" class="closebtn" onclick="closeNav()">&times;</a>
+      <div class="overlay-content">
+        <a href="#Portfolio" onclick="closeNav()">Portfolio</a>
+        <a href="#About" onclick="closeNav()">About</a>
+        <a href="#Contact" onclick="closeNav()">Contact</a>
+      </div>
```
Comment (andersonlebon) [inline index.html]: Here also the same 👇 
- [ ] Nice job 👏  👏  On implementing the mobile menu functionality I can see it is working properly, however one of the requirements says that you should use Javascript best practices. I think your project will be well structured if you put all Javascript functionality(events) in your `main.js file`. Kindly remove these `onclick` events here in your HTML and use `addEventListener in `main.js`
#### an example in your main.js
```js
    document.querySelector('.closebtn').addEventListener('click', closeNav)
```

**Review by andersonlebon — CHANGES_REQUESTED** [review]
## Hi @Thermiee 

Good job so far! I know you have worked hard on making this milestone But  There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there
## To highlight :champagne: 
- Good commit history :heavy_check_mark: 
- Your design looks good :100: :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
## Required Changes ♻️

- [ ] Nice job :clap: using `Javascript` for implementing this mobile menu feature, Hovever one of the requirements says that you should set up both JavaScript and HTML&CSS linters in the project repository.  Kindly follow [these](https://github.com/microverseinc/linters-config/tree/master/html-css-js) instructions for both linters configuration. You are missing the Javascript linters checker, see here 👇  down 

Your configuration | the better configuration with Js
------|-----
image | image


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mmsesay — APPROVED** [review]
Hi @Thermiee , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**